sp800-63c/sec5_strength.md

```diff
@@ -70,11 +70,11 @@ All assertions SHOULD use audience restriction techniques to allow an RP to reco
 
 #### 5.2.5. Pairwise Pseudonymous Identifiers
 
-In some circumstances, it is desirable to prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier. Pairwise pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP, and the CSP SHALL generate a different identifier for each RP.
+In some circumstances, it is desirable to prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier. Pairwise pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP, and the CSP SHALL generate a different identifier for each RP or integration with an RP, as appropriate.
```

Comment:
What does "integration with an RP" mean? Do we mean collections of related RPs, as referenced above in the "one or more" statement?